{{cookiecutter.github_repository}}/provisioner/roles/project_data/tasks/webpack.yml

```diff
@@ -13,11 +13,24 @@
   shell: apt update
   become: yes
 
+- name: Install Node dependencies for Ubuntu
+  become: yes
+  apt:
+    pkg: "{{ item }}"
+    state: present
+  with_items:
+    - libssl1.0-dev
+    - node-gyp
+    - npm
+
 - name: Install Node.js
   apt:
     pkg: nodejs
     state: present
 
+- name: Install webpack
+  shell: npm i webpack@4.31.0
```

Comment:
Can you try to update location of webpack in package.json.

replace `webpack` with `./node_modules/.bin/webpack`, in `"build"` script.

Reply:
Let me try this out on the server. Will get back to you on this.

Reply:
@theskumar As we discussed, the task before this one fails in that case:

```TASK [project_data : Install packages based on package.json.] ********************************
fatal: [dj-init.fueled.engineering]: FAILED! => {
    "changed": false
}

MSG:

Failed to find required executable npm in paths: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin

    to retry, use: --limit @/Users/sanyamkhurana/personal/django-init-tests/hello-world-web/provisioner/site.retry

PLAY RECAP ***********************************************************************************
dj-init.fueled.engineering : ok=2    changed=0    unreachable=0    failed=1```

So, let me know if there are any changes needed in this PR.